src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -144,17 +144,23 @@ const DateTimeBlock : React.FC<IProps> = ({
   }, [date, displayMinutes, setDateCallback]);
 
   const setDateMinutes = useCallback(({target: {value}}: React.ChangeEvent<HTMLInputElement>) => {
-    const minuteRegex = /^[0-5]{0,1}[0-9]{0,1}$/;
-    if (!minuteRegex.test(value)) {
-      return;
+    let newVal = '0';
+
+    if(Number(value) > 59){
+      newVal = '0';
```

Comment:
I think this should be '00'.

Reply:
Done, thanks

Reply:
Please note that, to fix this I have updated logic that whenever user hit backspace event, it will clear both the digit in the minute input field.